kube-derive/tests/crd_schema_test.rs

```diff
@@ -132,7 +132,7 @@ fn test_serialized_matches_expected() {
             nullable: None,
             nullable_skipped_with_default: None,
             nullable_with_default: None,
-            timestamp: DateTime::from_utc(NaiveDateTime::from_timestamp(0, 0), Utc),
+            timestamp: DateTime::from_utc(NaiveDateTime::from_timestamp_opt(0, 0).unwrap(), Utc),
```

Comment:
this looks sensible given the [deprecation warning](https://docs.rs/chrono/latest/chrono/naive/struct.NaiveDateTime.html#method.from_timestamp)